packages/backend-common/src/auth/createLegacyAuthAdapters.ts

```diff
@@ -168,39 +204,50 @@ class HttpAuthCompat implements HttpAuthService {
     req: Request,
     options?: {
       allow?: Array<TAllowed>;
-      allowedAuthMethods?: Array<'token' | 'cookie'>;
+      allowLimitedAccess?: boolean;
```

Comment:
should this one really ignore allowLimitedAccess? 🤔

Reply:
It can right now at least, since a limited access toke is just a regular one